python/pyspark/sql/connect/client.py

```diff
@@ -36,6 +41,49 @@
 )
 
 
+def _configure_logging() -> logging.Logger:
+    """Configure logging for the Spark Connect clients."""
+    logger = logging.getLogger("pyspark.sql.connect.client")
```

Comment:
```suggestion
    logger = logging.getLogger(__name__)
```

per https://docs.python.org/3/library/logging.html#logger-objects